pkgs/applications/office/espanso/default.nix

```diff
@@ -1,69 +1,89 @@
 { lib
 , fetchFromGitHub
 , rustPlatform
+, cargo-make
 , pkg-config
 , extra-cmake-modules
+, dbus
 , libX11
 , libXi
 , libXtst
 , libnotify
+, libxkbcommon
 , openssl
 , xclip
 , xdotool
+, setxkbmap
+, wl-clipboard
+, wxGTK32
 , makeWrapper
 , stdenv
 , AppKit
 , Cocoa
 , Foundation
+, waylandSupport ? true,
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "espanso";
-  version = "0.7.3";
+  version = "2.1.8";
 
   src = fetchFromGitHub {
-    owner = "federico-terzi";
+    owner = "espanso";
     repo = pname;
     rev = "v${version}";
-    sha256 = "1q47r43midkq9574gl8gdv3ylvrnbhdc39rrw4y4yk6jbdf5wwkm";
+    sha256 = "06wzrjdvssixgd9rnrv4cscbfiyvp5pjpnrih48r0ral3pj2hdg5";
   };
 
-  cargoSha256 = "0ba5skn5s6qh0blf6bvivzvqc2l8v488l9n3x98pmf6nygrikfdb";
+  cargoSha256 = "sha256-U2ccF7DM16TtX3Kc4w4iNV4WsswHJ0FpO3+sWCL1Li8=";
 
   nativeBuildInputs = [
     extra-cmake-modules
     pkg-config
     makeWrapper
+    cargo-make
+    wxGTK32
```

Comment:
Is this a nativeBuildInput?

Reply:
I've tried building without wxGTK32 as a nativeBuildInput, but the build failed.

<details>

```
       > error: failed to run custom build command for `espanso-modulo v0.1.0 (/build/source/espanso-modulo)`
       >
       > Caused by:
       >   process didn't exit successfully: `/build/source/target/release/build/espanso-modulo-8653a240257a137b/build-script-build` (exit status: 101)
       >   --- stderr
       >   thread 'main' panicked at 'wxWidgets is not installed, as `wx-config` cannot be executed', espanso-modulo/build.rs:467:5
       >   note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
       > warning: build failed, waiting for other jobs to finish...
       For full logs, run 'nix log /nix/store/dnn037vxl868icgfyimfwfcb604x1zfy-espanso-2.1.8.drv'.
```
</details>